_articles/it/best-practices.md

```diff
@@ -0,0 +1,314 @@
+---
+lang: it
+title: Buone pratiche per Maintainer
+description: Come rendere la tua vita da maintainer open source più facile. Dai processi di documentazione alla valorizzazione e utilizzo della tua community.
+class: best-practices
+toc:
+  what-does-it-mean-to-be-a-maintainer: "Cosa significa essere un maintainer?"
+  documenting-your-processes: "Il processo di documentazione"
+  learning-to-say-no: "Imparare a dire di no"
+  leverage-your-community: "Utilizza la tua community"
+  bring-in-the-robots: "Automatizza"
+  its-okay-to-hit-pause: "Prenditi una pausa"
```

Comment:
The `toc` section here should be removed as per point 5 in our [Contribution guidelines](https://github.com/github/opensource.guide/blob/e2c0ae140e50cfa83b8a5afeb4a6fd2c26d75910/docs/translations.md#starting-a-translation)